first draft commit

MotNosniktaw opened the pull request.

MotNosniktaw committed:
first draft commit

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,224 @@
+    <link rel="stylesheet" type="text/css" href="normalize-8-0-1.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Barlow|Cairo|Catamaran|Maven+Pro|Rajdhani&display=swap" rel="stylesheet">
+    <title>T-Dawg</title>
```
This will be a professional website

sjr-mayden replied:
Done

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,224 @@
+    <link href="https://fonts.googleapis.com/css?family=Barlow|Cairo|Catamaran|Maven+Pro|Rajdhani&display=swap" rel="stylesheet">
+    <title>T-Dawg</title>
+</head>
+<body id="Top">
```
Capitalised ID, should all be lowercase

sjr-mayden replied:
Done

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,224 @@
+            <div class="social-buttons">
+                <a href="https://twitter.com"><img class="social-button" src="img/twitter-button.png"></a>
+                <a href="https://github.com"><img class="social-button" src="img/github-button.svg"></a>
+                <a href="https://linkedin.com"><img class="social-button" src="img/linkedin-button.png"></a>
```
Get svgs whenever possible

sjr-mayden replied:
Done

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,224 @@
+        </div>
+        <div class="nav-mobile-bottom">
+            <div class="nav-links-mobile">
+                <a href="#About">About Me</a>
```
Capitals on IDs

sjr-mayden replied:
Done

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,224 @@
+                <p>
+                    Howdy! I'm Tom!<br>
+                    I am a Trainee Developer at the Mayden Academy. I am from Southampton in the UK and currently based in Bath.<br>
+
```
Extra newline

sjr-mayden replied:
Done

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+html {
+    scroll-behavior: smooth;
+}
+/* add conditional CSS for different*/
```
Wouldn't leave comments detailing a todo item in code submitted for pull request

Sounds like this needs to be completed before merging to master

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+}
+
+.about-container {
+    /*float: left;*/
```
Looks like this was commented to remove it - should be removed before merging with master

sjr-mayden replied:
Done

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+.port-item img {
+    width: 90%;
+    margin-top: 15px;
+
```
Extra newline

sjr-mayden replied:
Done

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+    height: 5vh;
+    border: solid 1px rgb(50, 50, 50);
+    background-color: rgb(131, 105, 56);
+/*
```
As above, commented code

sjr-mayden replied:
Done

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+}
+
+.nav-links-mobile a {
+     font-size: 1.3rem;
```
Weird indentation

sjr-mayden replied:
Done

sjr-mayden commented on styles.css:
```diff
@@ -0,0 +1,349 @@
+
+
+/*
+font-family: 'Maven Pro', sans-serif;
```
As above commented, commented code

sjr-mayden replied:
Done

MotNosniktaw committed:
added images to repos

MotNosniktaw committed:
added .sgvs to repos
added .svgs to html
reduced silliness
small cased IDs

MotNosniktaw committed:
- Indentation/Commenting housekeeping
- Change Social Media Buttons pngs for svgs
- Removed placeholder text/titles
- removed capitals from ID names

MotNosniktaw committed:
made edits to HEAD section for design purposes

MotNosniktaw committed:
remove placeholder phone number

MotNosniktaw committed:
removed erroneous line break in media query
